docs/source/_themes/sphinx_rtd_theme/static/css/theme.css

```diff
@@ -1,4 +1,4 @@
-﻿@charset "UTF-8";
```

Comment:
Don't you commit this from ipfabric by mistake? I afraid this will paint st2 docs brocade colors :)